extension/agenthealth/handler/useragent/useragent.go

```diff
@@ -78,6 +79,15 @@ func (ua *userAgent) SetComponents(otelCfg *otelcol.Config, telegrafCfg *telegra
 		ua.outputs.Add(output.Config.Name)
 	}
 
+	// Adding SELinux status
+	if os.Getenv("SELINUX") == "true" {
```

Comment:
We need to be consistent with consts - https://github.com/aws/amazon-cloudwatch-agent/blob/fbacc48316a66b73977c01e1891d69229e7b0a2e/cfg/envconfig/envconfig.go#L43

Reply:
Define a const. Look how and where `RUN_WITH_IRSA` is defined.

Reply:
nit: Probably call this `RUN_WITH_SELINUX`.